schema/sb.openapi.yaml

```diff
@@ -0,0 +1,79 @@
+---
+openapi: 3.0.3
+info:
+  title: Storage Broker API
+  description: Storage Broker API returns url to download AWS archive
+  version: 1.0.0
+
+paths:
+  /archive/url:
+    get:
+      summary: Returns archive download url
+      description: Fetches an archive download url from AWS.
+      operationId: api.archive.fetch
+      parameters:
+      - $ref: '#/components/parameters/requestId'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ArchiveInfo'
+        '400':
+          description: '#/components/responses/BadRequest'
+
+components:
+  schemas:
+    ArchiveInfo:
+      type: object
+      properties:
+        request_id:
+          $ref: '#/components/schemas/RequestId'
+        url:
+          $ref: '#/components/schemas/Url'
+        timeout:
```

Comment:
Is there any reason why timeout is not required as well?  IIRC this is something that we were looking to keep track of but I could be wrong about that.

Reply:
Good catch! Updated :+1: